### DISCUSSION TITLE
Ukulele Support

### DISCUSSION THREAD

**Discussion by danmazz2001** [CoderLine/alphaTab#592] [Ideas]
Ukulele has exploded in popularity over the last 10 years, surpassing guitar usage here in the US. Uke is a great instrument for children to start with since it is a bit smaller.  Our program Moosiko helps users learn guitar and ukulele (Soprano with tuning GCEA) and wanted to know when you plan on supporting ukulele?

**Comment by Danielku15** [comment]
Can you maybe explain a bit more what makes "ukulele" special compared to all the other stringed instruments using tab/standard notation? alphaTab has support for different string-variations, tunings and midi instruments and should support Ukulele perfectly fine. Of course unless Ukulele might have some very special own custom notation where I would need to get some more material on what you're referring to.

**Comment by danmazz2001** [comment]
Hi Daniel - I didn't know we could adjust the number of strings and
tuning of those strings. The main difference is that a uke has 4 strings
with the following tuning: G - C - E - A. Unlike a guitar, the top string
(string 4) is the lowest:

G3 (Low G) 196 Hz (top string)
C4 261.63 (often rounded to 262) Hz
E4 329.63 (often rounded to 330) Hz
A4 440 Hz (bottom string)

I'll be very happy if AlphaTab can already support this!

Dan

On Mon, Apr 19, 2021 at 4:47 PM Daniel Kuschny ***@***.***>
wrote:

> Can you maybe explain a bit more what makes "ukulele" special compared to
> all the other stringed instruments using tab/standard notation? alphaTab
> has support for different string-variations, tunings and midi instruments
> and should support Ukulele perfectly fine. Of course unless Ukulele might
> have some very special own custom notation where I would need to get some
> more material on what you're referring to.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CoderLine/alphaTab/discussions/592#discussioncomment-632172>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ2W3YWJQGF2ZD5RCZWURYTTJSJFDANCNFSM43FYLVHQ>
> .
>


-- 
*Dan*

**Reply by Danielku15** [reply]
I also have an Ukulele so I'm perfectly aware of the Instrument itself 😉 I just wasn't aware of any potential "special Ukulele Notations". AlphaTab supports any kind of string variations (tunings in alphaTab). If you use Guitar Pro as input, you get all the things out of the box. Just create a track with the ukulele preset and alphaTab will pick it up. If you use alphaTex, simply just define 4 strings in the [`\tuning`](https://alphatab.net/docs/alphatex/metadata#tuning) metadata. 

![image](https://user-images.githubusercontent.com/674916/115429760-01480c80-a204-11eb-91e6-d4a639a4c551.png)

**Comment by danmazz2001** [comment]
Excellent...thank you Daniel!

On Tue, Apr 20, 2021 at 12:13 PM Daniel Kuschny ***@***.***>
wrote:

> I also have an Ukulele so I'm perfectly aware of the Instrument itself 😉
> I just wasn't aware of any potential "special Ukulele Notations". AlphaTab
> supports any kind of string variations (tunings in alphaTab). If you use
> Guitar Pro as input, you get all the things out of the box. Just create a
> track with the ukulele preset and alphaTab will pick it up. If you use
> alphaTex, simply just define 4 strings in the \tuning
> <https://alphatab.net/docs/alphatex/metadata#tuning> metadata.
>
> [image: image]
> <https://user-images.githubusercontent.com/674916/115429760-01480c80-a204-11eb-91e6-d4a639a4c551.png>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/CoderLine/alphaTab/discussions/592#discussioncomment-636136>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJ2W3YRFKTY7OULZUZPFXA3TJWRZ5ANCNFSM43FYLVHQ>
> .
>


-- 
*Dan*
